config/routes.js

```diff
@@ -72,6 +72,12 @@ module.exports = function(app, passport, auth) {
     //Finish with setting up the userId param
     app.param('userId', users.user);
 
+    // Game routes
+    var game = require('../app/controllers/game');
+    app.get('/api/games', game.all);
+    app.get('/api/games/:id/start', users.authToken, game.start);
```

Comment:
@andela-jkanyua please note that `/api/games/:id/start` is an authenticated endpoint that requires a JWT token. Probably why it was not working. Thank you for pointing this out though, I have added a much more descriptive step-by-step explanation of how to obtain the token and how to use it. I have also replaced the screenshot with the deployed version.